docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -4,63 +4,391 @@ platform: platform
+{{% /warning %}}
+
+---
+## <a id="mainenvironmententerprise"></a> Enterprise-scale environment
+
+After adding your custom app-specific actions, you should now be ready to run the required tests for the Marketplace Data Center Apps Approval process. To do this, you'll need an **enterprise-scale environment**.
+
+### <a id="instancesetup"></a>5. Set up an enterprise-scale environment Jira Data Center on AWS
+
+We recommend that you use the [AWS Quick Start for Jira Data Center](https://aws.amazon.com/quickstart/architecture/jira/) to deploy a Jira Data Center enterprise-scale environment. This Quick Start will allow you to deploy Jira Data Center with a new [Atlassian Standard Infrastructure](https://aws.amazon.com/quickstart/architecture/atlassian-standard-infrastructure/) (ASI) or into an existing one.
 
 The ASI is a Virtual Private Cloud (VPC) consisting of subnets, NAT gateways, security groups, bastion hosts, and other infrastructure components required by all Atlassian applications, and then deploys Jira into this new VPC. Deploying Jira with a new ASI takes around 50 minutes. With an existing one, it'll take around 30 minutes.
 
-### Using the AWS Quick Start for Jira
+#### Using the AWS Quick Start for Jira
 
 If you are a new user, perform an end-to-end deployment. This involves deploying Jira into a _new_ ASI.
 
-If you have already deployed the ASI separately by using the [ASI Quick Start](https://aws.amazon.com/quickstart/architecture/atlassian-standard-infrastructure/)ASI Quick Start or by deploying another Atlassian product (Jira, Bitbucket, or Confluence Data Center), deploy Jira into your existing ASI.
+If you have already deployed the ASI separately by using the [ASI Quick Start](https://aws.amazon.com/quickstart/architecture/atlassian-standard-infrastructure/)ASI Quick Start or by deploying another Atlassian product (Jira, Bitbucket, or Confluence Data Center development environment), deploy Jira into your existing ASI.
 
 {{% note %}}
 You are responsible for the cost of the AWS services used while running this Quick Start reference deployment. There is no additional price for using this Quick Start. For more information, go to [aws.amazon.com/pricing](https://aws.amazon.com/ec2/pricing/).
 {{% /note %}}
 
 To reduce costs, we recommend you to keep your deployment up and running only during the performance runs.
 
-### AWS cost estimation ###
+#### AWS cost estimation
 [AWS Pricing Calculator](https://calculator.aws/) provides an estimate of usage charges for AWS services based on certain information you provide.
-Monthly charges will be based on your actual usage of AWS services, and may vary from the estimates the Calculator has provided.
+Monthly charges will be based on your actual usage of AWS services and may vary from the estimates the Calculator has provided.
+
+*The prices below are approximate and may vary depending on such factors like region, instance type, deployment type of DB, and other.  
```

Comment:
How about "ec2 instance type, db instance type"

Reply:
This is a good idea, however, this syntax construction adds some boundaries that we can overstep somehow.